gluon/gluon-luci-wifi-config/files/usr/lib/lua/luci/model/cbi/admin/wifi-config.lua

```diff
@@ -0,0 +1,90 @@
+local f, s, o
+local uci = luci.model.uci.cursor()
+local config = 'wireless'
+ 
+--set the heading, button and stuff 
+f = SimpleForm("wifi", "WLAN-Config")
+f.reset = false
+f.template = "admin/expertmode"
+f.submit = "Speichern"
+
+-- text, which describes what the package does to the user
+s = f:section(SimpleSection, nil, [[
+Viele Freifunk-Communitys betreiben ein sogenanntes Dachnetz. Das bedeutet, dass
+manche Router sich über große Strecken miteinander verbinden, um viele kleine
+Mesh-Netze miteinander zu verbinden. Um diese weiten Verbindungen effektiver zu
+gestalten hast du hier die Möglichkeit die SSID, mit der sich die Clients verbinden,
+zu deaktivieren.
+]])
+ 
+ 
+local radios = {}
+
+-- look for wifi interfaces and add them to the array
+uci:foreach('wireless', 'wifi-device',
+function(s)
+	table.insert(radios, s['.name'])
+end
+)
+
+--add a client and mesh checkbox  for each interface
+for index, radio in ipairs(radios) do
+ 	--get the hwmode to seperate 2.4GHz and 5Ghz radios
+	local hwmode = uci:get('wireless', radio, 'hwmode')
+	
+	if hwmode == '11g' or hwmode == '11ng' then --if 2.4GHz
+	 	--box for the clientnet
+		o = s:option(Flag, 'clientbox' .. index, "2,4GHz Client Netz aktivieren")
+		o.default = (uci:get_bool(config, 'client_' .. radio, "disabled")) and o.disabled or o.enabled
+		o.rmempty = false
+		--box for the meshnet 
+		o = s:option(Flag, 'meshbox' .. index, "2,4GHz Mesh Netz aktivieren")
+		o.default = (uci:get_bool(config, 'client_' .. radio, "disabled")) and o.disabled or o.enabled
+		o.rmempty = false
+	 
+	elseif hwmode == '11a' or hwmode == '11na' then --if 5GHz
+		--box for the clientnet
+		o = s:option(Flag, 'clientbox' .. index, "5GHz Client Netz aktivieren")
+		o.default = (uci:get_bool(config, 'client_' .. radio, "disabled")) and o.disabled or o.enabled
+		o.rmempty = false
+		--box for the meshnet
+		o = s:option(Flag, 'meshbox' .. index, "5GHz Mesh Netz aktivieren")
+		o.default = (uci:get_bool(config, 'client_' .. radio, "disabled")) and o.disabled or o.enabled
+		o.rmempty = false
+	 
+	end
+end
```

Comment:
What do you mean by that? 
It was my intent to make it possible to deactivate each SSID, because I talked with some people and this is what we wanted. We shouldn't limit the options, just because we think its good/bad.